public/diagnostic.js

```diff
@@ -23,3 +23,21 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  orderPrice: Ember.computed('price', 'quantity', function(){
+    return this.get('price') * this.get('quantity');
+  })
+});
+
+const Cart = Ember.object.extend({
+  orders: [],
+  addToCart: function(order){
+    this.get('orders').addObject(order);
+  },
+  totalPrice: Ember.computed('orders', 'orders.@each.orderPrice', function(){
+    let orders = this.get('orders')
+  }){
```

Comment:
so here's one way, but there are near-countless ways to do this. Great start. Next time you feel like that drop a comment and just pseudo code out what you want it to do. Even if you don't get the code by submission, you'll have the problem broken down for you when you come back to it 👍